### PR TITLE
[Login popup] Login button is clickable

### LINKED ISSUE #73: [Login popup] Login button is clickable without entering any data in Email and Password field
**Environment:** Windows10 21H2, Chrome 112.0.5615.138
**Reproducible:** always

**Preconditions**
The user is on the client website

**Steps to reproduce**

1. Click Login button in the upper right corner of the header
2. Hover the mouse that the cursor fall on Login button and hover back outside
3. Click on Login button

**Expected result**
The Login button should be unclickable, nothing should happen

User story #32 , acceptance criteria 3

### PR DESCRIPTION
**Pull request by Nazar-Lysak**
…word [Login popup] Login button is clickable without entering any data in Email and Password field #73

### CODE REVIEW

**Review by docia — APPROVED** [review]

**Review by docia — APPROVED** [review]

**Review by m6xcom — APPROVED** [review]

**Review by ohanovdmytro — APPROVED** [review]

**Review by Olenka-Hryk — COMMENTED** [review]
Do you check the Password field as well? Because according to the requirements, the Login button should be disabled until data is entered in both the Email and Password fields.

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
I agree with @Olenka-Hryk about password checking. Please add it too.

File: src/containers/guest-home-page/login-form/LoginForm.jsx [hunk 1]
```diff
@@ -32,6 +32,11 @@ const LoginForm = ({
     openModal({ component: <ForgotPassword /> })
   }
 
+  const buttonValidate = (emailValue) => {
+    return emailValue?.length && data?.password?.length ? false : true
+  }
+
+  console.log()
```
Comment (Olenka-Hryk) [inline src/containers/guest-home-page/login-form/LoginForm.jsx]: Could you please remove the ```console.log()``` statement from this code?

File: src/containers/guest-home-page/login-form/LoginForm.jsx [hunk 2]
```diff
@@ -32,6 +32,10 @@ const LoginForm = ({
     openModal({ component: <ForgotPassword /> })
   }
 
+  const buttonValidate = (emailValue) => {
+    return emailValue?.length && data?.password?.length ? false : true
```
Comment (vlad-khrychov) [inline src/containers/guest-home-page/login-form/LoginForm.jsx]: Look at comment from sonar 
image
You can write smth like
```suggestion
    return !(emailValue?.length && data?.password?.length)
```

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**